controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1525,9 +1521,43 @@ bool master_thread::handle_non_intel_slave_join(Socket *sd,
                                                 std::string bridge_mac, std::string radio_mac,
                                                 ieee1905_1::CmduMessageTx &cmdu_tx)
 {
-    // TODO backhaul handling.
-    // Multi-AP Agent doesn't say anything about the backhaul, so we have to rely on Intel Slave Join for that.
-    std::string backhaul_mac = "";
+
+    // Multi-AP Agent doesn't say anything about the backhaul, so simulate ethernet backhaul to satisfy
+    // network map.
+    sMacAddr mac = network_utils::mac_from_string(bridge_mac);
+    mac.oct[5]++;
+    std::string backhaul_mac     = network_utils::mac_to_string(mac);
+    std::string parent_bssid_mac = network_utils::ZERO_MAC_STRING;
+    std::string manufacturer(tlvwscM1->manufacturer(), tlvwscM1->manufacturer_length());
+
+    LOG(INFO) << "IRE generic Slave joined, sd=" << intptr_t(sd) << std::endl
+              << "    manufacturer=" << manufacturer << std::endl
+              << "    parent_bssid_mac=" << parent_bssid_mac << std::endl
+              << "    al_mac=" << bridge_mac << std::endl
+              << "    backhaul_mac=" << backhaul_mac << std::endl
+              << "    radio_identifier = " << radio_mac << std::endl;
+
+    LOG(DEBUG) << "simulate backhaul connected to the GW's LAN switch ";
+    auto gw_container = database.get_nodes_from_hierarchy(0, beerocks::TYPE_GW);
```

Comment:
The first commit does some whitespace mess-up which is fixed by the second commit. Please do it correctly in the first commit while rebasing.

Reply:
Fixed